apps/scorecard/.gitignore

```
@@ -0,0 +1,4 @@
+node_modules
+.meteor/local
+.idea
+settings
```

`node_modules` and `.idea` are covered by the [`.gitignore` in the repo root](https://github.com/NOAA-GSL/MATS/blob/development/.gitignore) so they shouldn't need to be repeated here.

If  we generally wanted to ignore `.meteor/local` and `settings` as well, we could add them to the root `.gitignore`. This would let us remove the extra per-app `.gitignore` files.

```suggestion
.meteor/local
settings

```

---

Agreed, much of this should be covered by the .gitignore in the repo root.

---

Good point, Molly is making an issue to address this, but it isn't a requirement for this pull request. Good catch.

---

Sounds good. I've captured this in #790. @mollybsmith-noaa - based on the comments elsewhere in the PR should I assign it to you?